src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTargetSpelling.java

```diff
@@ -39,6 +45,12 @@
 
 public class TextEditingTargetSpelling implements TypoSpellChecker.Context
 {
+   interface Resources extends ClientBundle
+   {
+      @Source("../../../../commands/goToWorkingDir.png")
```

Comment:
Since these icons are semantically different I think they should be different files (even if they are visually identical to begin with!). Would recommend making a copy of the PNG with an appropriate name and placing it in this folder so you don't need to reach outside the directory to form the bundle.